web-app/README.md

```diff
@@ -1,3 +1,195 @@
+
+**Make a new proxy route in Notehub**
+
+Once the Notehub project exists, inside of the project's **Routes** tab, click the **Create Route** button in the upper right hand corner of the screen, and select a **Proxy** route type.
+
+![Choose to create a new proxy Notehub route](images/notehub-create-proxy.png)
+
+**Get the URL for Notehub from the AWS ZIP file**
+
+Go back to the AWS S3 bucket with the ZIP file and click on the file name to see its information. Here in the kiosk.zip page, copy the **Object URL** - this is what goes into the **URL** input of the Notehub route.
+
+![Copy the Object URL from the AWS ZIP file's info](images/aws-bucket-url.png)
+
+**Paste the URL into the Notehub route and give the route an alias**
+
+Back over in the Notehub route, paste the AWS URL into the **URL** input and remove the `/kiosk.zip` from the end of the string. Give the route an alias - something like `kiosk`. This alias must match what's in the Notehub environment variable `kiosk_content`, and it's how the Python script will know which ZIP file to download if there are multiple ZIP files contained within the same S3 bucket.
+
+![Paste the Object URL into the Notehub route and give it an alias](images/notehub-proxy.png)
+
+## Make Sure the Notehub Fleet Environment Variables are Set
+
+Once again, if you were following along with the [main `README.md` for this project](https://github.com/blues/app-accelerators/blob/main/27-cellular-connected-electronic-kiosk/README.md), you're aware that there are 4 Notehub environment variables needed for this application.
+
+* `kiosk_content` - indicates the ZIP file to be downloaded from the Notehub proxy route.
+* `kiosk_content_version` - a version number for the file specified by `kiosk_content`. If the content's updated, this number can be incremented to cause the Python script to download the new content.
+* `kiosk_download_time` - the hour of the day for the Python script to check for updates to the `kiosk_content` env var. It can also be set to `now` to download as soon as new content is detected either by changing the `kiosk_content` variable or the `kiosk_content_version` number.
+* `kiosk_data` - An optional JSON object that can be used to overwrite the contents of the `data.js` in the ZIP file. 
+
+Keep in mind, that however the code in `data.js` is structured, the JSON in this `kiosk_data` environment variable must mimic it. For example, in the code, `data.js` looks like this:
+
+```javascript
+var data = {
+  cities: ["Barcelona", "Madrid", "Dublin", "Tokyo"],
+};
+```
+
+So for the `kiosk_data` Notehub environment variable, I created the following JSON object:
+
+```json
+{"cities": ["San Fernando", "Port of Spain", "Atlanta", "Shen Zhen"]}
```

Comment:
Are these cities supposed to match the ones above?

Reply:
No, they're supposed to differ from what's zipped up with the web app by default to help illustrate how changing them will affect the cities displayed in the browser without actually modifying the ZIP file's code.